controllers/dashboardRoutes.js

```diff
@@ -0,0 +1,38 @@
+const { ProjectEmployee, Project, Task } = require('../models');
+
+const router = require('express').Router();
+
+router.get('/', async (req, res) => {
+    const projectEmployee = await ProjectEmployee.findAll({where: {employee_id: req.session.userId}});
+    const projects = [];
+    const tasks = await Task.findAll({where: {employee_id: req.session.userId}});
+    for (i in projectEmployee) {
+        const project = await Project.findAll({where: {id: projectEmployee[i].dataValues.id}});
+        projects.push(project);
+    };
+    res.render('dashboard', {projects, tasks, isMgr: req.session.mgr})
```

Comment:
we needs to get the plain data add the below line above the res.render line.
const projects = await projectData.map((project) => project.get({ plain: true }));